ctapipe/image/timing_parameters.py

```diff
@@ -0,0 +1,87 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+# -*- coding: UTF-8 -*-
+"""
+Image timing-based shower image parametrization.
+"""
+
+from collections import namedtuple
+import numpy as np
+from astropy.units import Quantity
+
+__all__ = [
+    'TimingParameters',
+    'timing_parameters'
+]
+
+TimingParameters = namedtuple(
+    "TimingParameters",
+    "gradient, intercept"
+)
+
+
+class TimingParameterizationError(RuntimeError):
+    pass
+
+
+def rotate_translate(pixel_pos_x, pixel_pos_y, phi):
```

Comment:
Probably this should go somewhere more common at some point. It's also faster to use a matrix multiplication and the `utils.linalg.rotation_matrix_2d`